CuratedContent/ManualWorkIsABug.md

```diff
@@ -0,0 +1,72 @@
+
+## Manual Work is a Bug
+
+<!-- deck text start -->
+Work to automate routine computing processes using an Agile incremental approach, starting by documenting the manual steps and then incrementally automating pieces of the process using scripts/code.
+Continuing to do routine processes repeatedly with manual execution is a bug!
+<!-- deck text end -->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe "Roscoe A. Bartlett")
+#### Publication date: May 27, 2024
+
+Resource information | Details
+:--- | :---
+Paper Title | Manual Work is a Bug
+Authors | Thomas A. Limoncelli
+Publication | [ACM Queue, March 14, 2018, Volume 16, Issue 1](https://queue.acm.org/detail.cfm?id=3197520)
+
+Nearly everyone who works with computers often has to perform routine tasks.
+Some people like system administrators (sysadmins) and DevOps engineers have **many** such tasks to perform.
+However, even other types of developers and users often have repetitive tasks that they need to execute repeatedly.
+And others in their teams and organizations often need to perform similar processes/tasks (or the same tasks, with minor variations).
+These routine processes may take long enough, be complex enough, have severe enough consequences if they are done in correctly, and/or be likely to be repeated enough to justify automating them to some degree.
```

Comment:
```suggestion
These routine processes may be time-consuming, be complex, have severe consequences if they are done incorrectly, and/or be likely to be repeated enough to justify automating them to some degree.
```

Reply:
I was trying to emphasize that there my be thresholds for time, complexity, severity, and/or future repeatability to trigger considering automating the process.   Does the updated text make that clear?

Reply:
Applied in commit 34a11ccbd84db1ea44bcf3eaea9c31c584cfe4fa (except I removed "be" from "be complex").